internal/npm_install/generate_build_file.ts

```diff
@@ -571,53 +597,51 @@ export function getDirectDependencySet(pkgJsonPath: string): Set<string> {
 /**
  * Finds and returns an array of all packages under a given path.
  */
-function findPackages(p: string, dependencies: Set<string>) {
-  if (!isDirectory(p)) {
-    return [];
+async function findPackages(p: string, dependencies: Set<string>, pkgs: Array<Dep>) {
```

Comment:
Can we just `return await Promise.all` more like previously instead of passing the array in?